Makefile

```diff
@@ -440,3 +446,11 @@ test-e2e: defaults manifests generate fmt vet ## Run e2e tests.
 .PHONY: setup-e2e
 setup-e2e: ## Set up e2e tests.
 	KUBERAY_VERSION=$(KUBERAY_VERSION) test/e2e/setup.sh
+
+.PHONY: imports
+imports: openshift-goimports ## Organize imports in go files using openshift-goimports. Example: make imports
+	$(OPENSHIFT-GOIMPORTS)
+
+.PHONY: verify-imports
+verify-imports: openshift-goimports ## Run import verifications.
+	./hack/verify-imports.sh $(OPENSHIFT-GOIMPORTS)
```

Comment:
We can inline the logic here if we want to.

Reply:
I wasn't able to inline the logic in a way that looked clean, so I'd prefer to leave it in the script for now.

Reply:
Sounds good, not a big deal.